docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,194 @@ The parent key for all of the following parameters is `appuio_cloud_reporting`.
 
 [horizontal]
 type:: string
-default:: `syn-appuio-cloud-reporting`
+default:: `appuio-cloud-reporting`
 
 The namespace in which to deploy this component.
 
 
+== `images`
+
+[horizontal]
+type:: dictionary
+default::
++
+[source,yaml]
+----
+reporting:
+  registry: ghcr.io
+  repository: appuio/appuio-cloud-reporting
+  tag: ...
+
+erp_adapter:
+  registry: ghcr.io
+  repository: vshn/appuio-odoo-adapter
+  tag: ...
+----
```

Comment:
No point in listing image defaults in the docs since they generally get forgotten when updating the defaults, just link to `class/defaults.yml`:

```suggestion
default:: https://github.com/appuio/component-appuio-cloud-reporting/blob/master/class/defaults.yml[See `class/defaults.yml].
```